### ISSUE TITLE
Cinnamon crash report

### ISSUE THREAD

**Issue by ahmed-elsayed2017** [linuxmint/cinnamon#9141]
```
*Cinnamon version: 4.4.8
   - Please specify if you are using the daily builds PPA
     I don't use it.
*Distribution: Linux Mint Cinnamon 19.3
*Graphics hardware, and driver used: AMD Radeon HD 8750M - Mesa 19.3.2 (kisak-mesa ppa)
*32 or 64 bit: 64bit.
```

[xsession-errors.txt](https://github.com/linuxmint/cinnamon/files/4114407/xsession-errors.txt)

**Issue**

I got this crash today, and my system froze, and I had to unplug my laptop from the power source.

*Here is Pastebin from System Reports:

[Pasetbin.txt](https://github.com/linuxmint/cinnamon/files/4114433/Pasetbin.txt)

**Comment by ahmed-elsayed2017** [comment]
Another crash happened today after pressing Qbittorrent icon in the taskbar.

[Pasetbin2.txt](https://github.com/linuxmint/cinnamon/files/4118843/Pasetbin2.txt)

**Comment by icarter09** [comment]
Cinnamon 4.4.8
Mint 19.3
64bit
Kernel 5.3.0-28

I downloaded Qbittorrent to see if that would cause any issues for me, but so far it hasn't. 

@ahmed-elsayed2017 have you experienced this issue since your latest comment?

**Comment by ahmed-elsayed2017** [comment]
I stopped using Qbittorrent, because it always keep checking the last downloaded torrent on every boot, and that cripples my machine.

Linux Mint freezes completely sometimes when I use the browser, or just playing a video, and I have to close my machine using the power button, and starting it again.

Anyway, I use now Kubuntu 19.10 to see if it will encounter the same issues, and I had the same issues, so I changed the kernel to an older kernel (kernel 4.19.101), and I didn't encounter any issues so far, but I am not quite sure if it will stay the same way, but I am optimistic.
